slam/surface_profiling.py

```diff
@@ -0,0 +1,354 @@
+
+import numpy as np
+import slam.geodesics
```

Comment:
PEP compliance about imports, slam import should be after numpy import separated by one line (pep8 ll deal with i)